TESTING.rst

```diff
@@ -196,6 +196,44 @@ Run all Quarantined tests:
 
      ./breeze --test-type Quarantined tests --db-reset
 
+Helm Unit Tests
```

Comment:
In my opinion, it is worth adding a command that runs the tests - `pytest chart/tests`.  All in all, this is the only thing that is important because the rest of the user can copy from other tests.

Reply:
Oof forgot htat line :)

Reply:
@mik-laj added